src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -622,7 +622,7 @@ private void ValidateGamesInTournamentSchemeTwo(Game newGame, List<GameResultDto
             }
         }
 
-        private void ValidateGamesInTournamentSchemeOne(Game newGame, List<GameResultDto> games)
+        private void ValidateGamesInTournamentSchemeOne(Game newGame, ICollection<GameResultDto> games)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<VolleyManagement.Domain.GamesAggregate.GameResultDto>' instead of 'System.Collections.Generic.ICollection<VolleyManagement.Domain.GamesAggregate.GameResultDto>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)

Reply:
?